jdk/src/java.base/windows/conf/amd64/jvm.cfg

```diff
@@ -1,3 +1,6 @@
+#
+# (c) Copyright IBM Corp. 2013, 2017 All Rights Reserved
```

Comment:
Format needs to be 'creation_date, modified_date', plus I believe you need to make this the same as OpenJ9.gmk.

Reply:
Andrew is on vacation. Checking other changes, the other modified files have the single line IBM copyright added, and use the same start date as the Oracle copyright. We can use the same pattern.

Reply:
Why are we adding a copyright to an otherwise unmodified file? Is there a change somewhere I'm not seeing?

Reply:
It's been modified in PR #65, forgot the copyright at that time.